tasks/main.yml

```diff
@@ -65,7 +38,7 @@
     group: 'root'
     mode: '0644'
   when: (item.value.state|d("present") != "absent" and
-         (item.value.type|d("default") not in ["divert"] or item.value.raw|d()))
+         (item.value.type|d("default") not in ["divert", "dont-create"] or item.value.raw|d()))
```

Comment:
I noticed the `ignore` and I agree with it, however, I already tried to name the option as good as possible and `dont-create` does exactly what it says. With this, snippets can still be enabled and disabled, but the role does not try to create the template for them. This is why I thought making it explicit by this name is preferred. `ignore` does not really hit the nail on the head in this case :wink: